bundles/org.eclipse.emfcloud.modelserver.client/src/org/eclipse/emfcloud/modelserver/client/ModelServerClientApiV1.java

```diff
@@ -22,7 +23,11 @@
 
    CompletableFuture<Response<A>> get(String modelUri, String format);
 
-   CompletableFuture<Response<List<String>>> getAll();
+   CompletableFuture<Response<List<Model<String>>>> getAll();
+
+   CompletableFuture<Response<List<Model<EObject>>>> getAll(String format);
```

Comment:
We should keep the generic return type `<A>` in this case, as the current `ModelServerClient` implements the `ModelServerClientApi<EObject>` anyway.